h/static/styles/loggedout-message.scss

```diff
@@ -0,0 +1,36 @@
+.loggedout-message {
+  margin: 25px auto;
```

Comment:
Where do these margins come from? This also duplicates the margin for the `.loggedout-message-logo` class. If there are two pieces of styling that are the same for a reason, please make use of variables and mixins to indicate this. Have a look at `excerpt.scss` for examples of this. Note that in that file, the purpose of the `@at-root` at the top is to prevent the variables from being used in other files.